src/Components/ToDoList/ToDoList.jsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component, useState } from 'react';
+import PropTypes, { bool, string } from 'prop-types';
+import './ToDoList.css';
+
+export default class ToDoList extends Component {
+  render() {
+    const {
+      todos,
+      check,
+      modifyItem,
+      deleteItem,
+    } = this.props;
+
+    return (
+      <ul className="ul-Container">
+        {todos.map((ObjecList) => (
+          <Item
+            key={ObjecList.id}
+            id={ObjecList.id}
+            title={ObjecList.title}
+            checked={ObjecList.completed}
+            addCheck={check}
+            modifyItem={modifyItem}
+            deleteItem={deleteItem}
+          />
+        ))}
+      </ul>
+    );
+  }
+}
+
+// Send the title and the crossline over the text
+const Item = (props) => {
```

Comment:
It is good to create a separate file for this component

Reply:
Good Idea, thank you for the advice!